kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/ProxyReconcilerIT.java

```diff
@@ -68,18 +68,18 @@ class ProxyReconcilerIT {
     private static final String CLUSTER_BAR_BOOTSTRAP = "my-cluster-kafka-bootstrap.bar.svc.cluster.local:9092";
     private static final String NEW_BOOTSTRAP = "new-bootstrap:9092";
 
-    private static KubernetesClient client;
+    private KubernetesClient client;
```

Comment:
It seems their test helper classes now close the Kube client, so we can't reuse a static one. Therefore we create a new one each time.